### DISCUSSION TITLE
用cloudfront workers 出现{"code":401,"message":"expire missing"}

### DISCUSSION THREAD

**Discussion by anyingxiuluo** [AlistGo/alist#3891] [Q&A]
我用的 https://github.com/alist-org/alist-proxy/blob/main/alist-proxy.js这里面的代码
ADDRESS是我的二级域名，端口80，token复制的设置-其他-令牌，出现{"code":401,"message":"expire missing"}
有没有大佬知道是什么原因。
![image](https://user-images.githubusercontent.com/125189022/226173668-1aa694f0-d055-46c7-a696-d580ef58ff76.png)

**Comment by M-CU** [comment]
请问cloudflare workers怎么用呀

**Comment by Guichen1210** [comment]
我也遇到这个问题，求助大佬解决方法

**Comment by jkoor** [comment]
workers 不是让你直接打开网站用的，是设置下载代理的

**Reply by Nirzak** [reply]
Well I was thinking the same. But then I found this one. https://alist.comy.workers.dev/ this one is also using alist. but he is directly opening the entire app using worker domain. not just the download urls.

**Reply by jkoor** [reply]
Cloudflare Workers supports reserve proxy, maybe this website use it. 

[cloudflare worker proxy](https://github.com/ymyuuu/Cloudflare-Workers-Proxy)
